### ISSUE TITLE
Demo videos to contribute

### ISSUE THREAD

**Issue by Tetsujinfr** [MCG-NKU/E2FGVI#9]
Hi,

Thanks for this great repo and project.

Not really an issue, more a question: 
I see the demo video section is TBD, would you be interested by some inferenced test videos in the wild for the read me?
I am planning to run some anyway, hopefully in the next week or so, let me know and I ll share.

Would be great to have higher res trained model to produce better quality demo videos too, but I see it is on the book of work.

**Comment by Paper99** [comment]
Thank you for your interest.
We expect to release the model, which can handle videos with arbitrary resolution, at May 15, 2022.

Would it solve your problem if we provided the model?

**Comment by Tetsujinfr** [comment]
@Paper99 that would be perfect yes. I will generate some demo videos with the current limited resolution anyway just for fun.

When you say "arbitrary resolution", your model is trained on multiple resolution training datasets and manages resolutions dynamically for the inpainting inference? 
Do you know the gpu memory usage of the model for a 4K video?

**Comment by Paper99** [comment]
Our "arbitrary resolution" model was still trained on videos with fixed resolution (432x240), but it generalizes well to much higher resolutions.
We found that directly completing a 720p video requires about 40G GPU memory.
Thus, for memory efficiency, we need to consider patch cropping strategy when we inference a high-resolution video.

**Comment by Tetsujinfr** [comment]
I did test on a few videos in the wild. It works well when the background is not systematically  occluded during the video sequence,  but for pseudo static sequence it struggle to remove large portions of the frame area, like in an interview type of video. It seems very sensitive to the accuracy of the masks provided, although the tennis example sequence seems to prove me wrong: not sure what to conclude on mask accuracy impact.
Also, it burns a lot of memory for long sequences (on my gpu, 5-7gb was used for a 5 seconds 30fps video.

Really cool work, I attach a good result test video I ran. Keen to test your high res model if I have enough gpu ram .

https://user-images.githubusercontent.com/29961693/167742361-0dd832c6-bc1f-4a59-8839-a03551991ec7.mp4
